games-action/supertuxkart/supertuxkart-1.2.recipe

```diff
@@ -33,15 +32,13 @@ REQUIRES="
  lib:libpng$secondaryArchSuffix
  lib:libsdl2_2.0$secondaryArchSuffix
  lib:libsqlite3$secondaryArchSuffix
-#lib:libsquish$secondaryArchSuffix
```

Comment:
libsquish has been merged and (checked) working on gcc2h, maybe enable it instead of removing it?

Reply:
Trimmed it for now. We can use SHARED later.

Reply:
This isn't related to SHARED (that only involves bundled libraries that are not available)